packages/loot-core/src/client/shared-listeners.ts

```diff
@@ -178,17 +197,20 @@ export function listenForSyncEvent(actions, store) {
           const { cloudFileId } = store.getState().prefs.local;
 
           notif = {
-            title: 'Syncing has been reset on this cloud file',
+            title: i18n.t('Syncing has been reset on this cloud file', {}),
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
            title: i18n.t('Syncing has been reset on this cloud file'),
```